src/platform/site-wide/accessible-VCL-modal.js

```diff
@@ -15,7 +15,9 @@ export function addFocusBehaviorToCrisisLineModal() {
   let openControl;
   const closeControl = tabbableElements[0];
   const lastTabbableElement = tabbableElements[tabbableElements.length - 1];
-  const triggers = document.querySelectorAll('[data-show="#modal-crisisline"]');
+  const triggers = Array.from(
```

Comment:
Should we create a helper for this and add to the platform utils?

Reply:
I think we probably need to refactor how this and the legacy/menu.js work together, and consolidate this code into something that works for any non-React modal we might create.

Reply:
Does this script get run through Babel? `Array.from` will break in IE11, which is too bad because it's so useful.

Reply:
Yep, any code in `src/` will get run through Babel.